src/main/resources/migrations/common/R__v_video_appointments.sql

```diff
@@ -13,7 +13,8 @@ select
   pa.prison_location_id,
   pa.appointment_date,
   pa.start_time,
-  pa.end_time
+  pa.end_time,
+  COALESCE(vlb.amended_time, vlb.created_time) as last_updated
```

Comment:
Hmm not sure on this, putting the created time into a field called last updated if last updated is null doesn't feel right.